recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -31,9 +40,6 @@ def validate(self):
         if not str(self.settings.arch) in ("x86_64"):
             raise ConanInvalidConfiguration(
                 f"{self.ref} is not supported yet.")
-        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) < "16":
-            raise ConanInvalidConfiguration(
```

Comment:
Why is this being dropped?

Reply:
The `check_min_vs` above already checks Visual Studio as well, I think.

Reply:
Ahh, was hidden! Thanks